vite.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
     alias: {
       // Alias src directory for @/components/{KongponentName} imports
       '@': path.resolve(__dirname, './src'),
+      mocks: path.resolve(__dirname, './mocks'),
```

Comment:
Since this is an alias, let's utilize `@` but that will require adjusting the other alias as well (you'll need to make sure this doesn't break anything)
```suggestion
      '@/': path.resolve(__dirname, './src/'),
      '@mocks/': path.resolve(__dirname, './mocks/'),
```